src/main/java/duke/task/Parser.java

```diff
@@ -0,0 +1,110 @@
+package duke.task;
+
+import duke.command.AddCommand;
+import duke.command.Command;
+import duke.command.FindCommand;
+import duke.command.DeleteCommand;
+import duke.command.ExitCommand;
+import duke.command.ListCommand;
+import duke.command.MarkCommand;
+import duke.command.UnmarkCommand;
+
+/**
+ * Class for Parser object.
+ *
+ * @author Gao Mengqi
+ * @version CS2103T AY22/23 Semester 2
+ */
+
+public class Parser {
+    private static String[] arr;
+    private static final String TODO = "todo";
```

Comment:
I like that 'magic' Strings are avoided here, as well as in other areas of your code.